<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>datos_tabulados</class>
 <widget class="QMainWindow" name="datos_tabulados">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>898</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(249, 247, 245)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabla_datos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>441</width>
      <height>141</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_graficar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;;
background-color: rgb(9, 139, 148)</string>
    </property>
    <property name="text">
     <string>Graficar columna seleecionada</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atras">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>800</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;;background-color: rgb(9, 139, 148)</string>
    </property>
    <property name="text">
     <string>Atras</string>
    </property>
   </widget>
   <widget class="QLabel" name="subtitulo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Escoger CSV:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_subir_CSV">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;;

background-color: rgb(9, 139, 148)</string>
    </property>
    <property name="text">
     <string>Subir CSV</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lista_columnas">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>431</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="name_archivo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:  15pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Archivo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="subtitulo_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:  15pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Selecione una columna para graficar:</string>
    </property>
   </widget>
   <widget class="QLabel" name="grafica">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>640</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imagenes/img proy inf/truncfinal01.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>logo</zorder>
   <zorder>tabla_datos</zorder>
   <zorder>boton_graficar</zorder>
   <zorder>atras</zorder>
   <zorder>subtitulo</zorder>
   <zorder>name_archivo</zorder>
   <zorder>subtitulo_2</zorder>
   <zorder>grafica</zorder>
   <zorder>boton_subir_CSV</zorder>
   <zorder>lista_columnas</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="logo_pricipal.qrc"/>
 </resources>
 <connections/>
</ui>
